bundles/binding/org.openhab.binding.sonance/src/main/java/org/openhab/binding/sonance/internal/SonanceBinding.java

```diff
@@ -185,18 +182,8 @@ protected void execute() {
         }
     }
 
-    protected void addBindingProvider(SonanceBindingProvider bindingProvider) {
-        super.addBindingProvider(bindingProvider);
-    }
-
-    protected void removeBindingProvider(SonanceBindingProvider bindingProvider) {
-        super.removeBindingProvider(bindingProvider);
-    }
-
```

Comment:
The above two methods are required for this service to run under openHAB 2.  Please restore them.